docs/apis-tools/spring-zeebe-sdk/configuration.md

```diff
@@ -8,9 +8,71 @@ This page uses YAML examples to show configuration properties. Alternate methods
 :::note
 Configuration properties can be defined as environment variables using [Spring Boot conventions](https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.relaxed-binding.environment-variables). To define an environment variable, convert the configuration property to uppercase, remove any dashes `-`, and replace any delimiters `.` with underscore `_`.
 
-For example, the property `camunda.client.zeebe.defaults.max-jobs-active` is represented by the environment variable `CAMUNDA_CLIENT_ZEEBE_DEFAULTS_MAXJOBSACTIVE`.
+For example, the property `camunda.client.worker.defaults.max-jobs-active` is represented by the environment variable `CAMUNDA_CLIENT_WORKER_DEFAULTS_MAXJOBSACTIVE`.
 :::
 
+## Modes
+
+The Spring SDK has modes with meaningful defaults aligned with the distribution's default connection details. Each mode is made for a Camunda 8 setup, and only one mode may be used at a time.
```

Comment:
My last question -- do I need to run a particular command to disable one mode in order to enable another given only one mode can be used at a time?

Reply:
This is a property being added to the application.yaml or added as env property as described above.